### PR TITLE
docs: extend plugin catalog

### PR DESCRIPTION
**Pull request by alexeagle**

### CODE REVIEW

File: docs/plugins/plugins.schema.json [hunk 1]
```diff
@@ -12,10 +12,40 @@
...
+          },
+          "homepage": {
+            "description": "A website describing the plugin. If not present, we'll use the GitHub UI",
+            "type": "string"
+          },
+          "name": {
+            "description": "The value users of the plugin put in the 'name' field of their .aspect/cli/plugins.yaml when using this plugin",
+            "type": "string"
+          },
+          "maintainers": {
```
Comment (alexeagle) [inline docs/plugins/plugins.schema.json:24]: note, I made this match the bazel-central-registry metadata.json schema just to reduce our overall cognitive load, may as well match the other place we publish stuff.

File: docs/plugins/plugins.d.ts [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+ * and run json-schema-to-typescript to regenerate this file.
+ */
+
+export interface AspectCLIPluginsCatalog {
+  $schema?: string;
+  plugins: {
+    /**
+     * GitHub repository
+     */
+    "github-repo": string;
```
Comment (kormide) [inline docs/plugins/plugins.d.ts:14]: On the registry I think they use `repository`.

Comment (alexeagle) [reply docs/plugins/plugins.d.ts:14]: yeah, but they support more than just github, and we don't yet. Hm. I was going to just barrel ahead but I think you're right, it's better to have `repository = github:some/repo` and then do the error handling/reporting

File: docs/plugins/plugins.d.ts [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+    /**
+     * Source code repository, starting with the host, e.g. github:my-org/some-repo
+     */
+    repository: string;
+    /**
+     * A website describing the plugin. If not present, we'll use the GitHub UI
+     */
+    homepage?: string;
+    /**
+     * The value users of the plugin put in the 'name' field of their .aspect/cli/plugins.yaml when using this plugin
```
Comment (gregmagolan) [inline docs/plugins/plugins.d.ts:20]: "value users"?

File: docs/plugins/plugins.schema.json (hunk 1 above)
Comment (gregmagolan) [reply docs/plugins/plugins.schema.json:24]: nice; looks good

**Review by gregmagolan — APPROVED** [review]

**Pull request merged**